src/v2/Components/__tests__/LotTimer.jest.tsx

```diff
@@ -96,13 +97,60 @@ describe("extendedBiddingInfoCopy", () => {
         "*Closure times may be extended to accomodate last minute bids"
       )
     })
+    describe("a bid has extended the auction", () => {
+      it("shows the extended next to the timer", () => {
+        let startDate = new Date()
+        const startAt = new Date(startDate.setMonth(startDate.getMonth() - 1))
+        let endDate = new Date()
+        const saleArtwork: LotTimer_saleArtwork = {
+          endAt: new Date(
+            endDate.setMinutes(endDate.getMinutes() + 1)
+          ).toISOString(),
+          formattedStartDateTime: "",
+          extendedBiddingEndAt: new Date(
+            endDate.setMinutes(endDate.getMinutes() + 3)
+          ).toISOString(),
+          sale: {
+            startAt: startAt.toISOString(),
+            extendedBiddingPeriodMinutes: 2,
+          },
+          " $refType": "LotTimer_saleArtwork",
+        }
+        const wrapper = mount(<LotTimer saleArtwork={saleArtwork} />)
+        const text = wrapper.text()
+        expect(text).toContain("Extended:")
+      })
+    })
+    describe("the auction has not yet been extended", () => {
+      it("does not show extended next to the timer", () => {
+        let startDate = new Date()
+        const startAt = new Date(startDate.setMonth(startDate.getMonth() - 1))
+        let endDate = new Date()
+        const saleArtwork: LotTimer_saleArtwork = {
+          endAt: new Date(
+            endDate.setMinutes(endDate.getMinutes() + 1)
+          ).toISOString(),
+          formattedStartDateTime: "",
+          extendedBiddingEndAt: "",
+          sale: {
+            startAt: startAt.toISOString(),
+            extendedBiddingPeriodMinutes: 2,
+          },
+          " $refType": "LotTimer_saleArtwork",
+        }
+        const wrapper = mount(<LotTimer saleArtwork={saleArtwork} />)
+        const text = wrapper.text()
+        expect(text).not.toContain("Extended: 2m0s")
```

Comment:
On the flip side, _this_ test is probably appropriate to ensure it _doesnt_ contain `Extended:` (and no need for the '2m0s' - it doesn't add anything to the spec).